File: java/src/main/java/devtools/handlebars/HandlebarsProperties.java
package devtools.handlebars;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;

@Getter
@Setter
@ConfigurationProperties(prefix = "spring.handlebars")
public class HandlebarsProperties {

    String prefix;
    String suffix;
    boolean cache;
    String delimiter = "{{,}}";

}

File: src/main/kotlin/devtools/handlebars/HandlebarsProperties.kt
package devtools.handlebars
import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.boot.context.properties.ConstructorBinding

@ConstructorBinding
@ConfigurationProperties(prefix = "spring.handlebars")
data class HandlebarsProperties (
        val prefix: String = "",
        val suffix: String = "",
        var cache: Boolean = false)
